src/language/branches.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Branches 
+description: Learn how to use branches to control the flow of your Dart code.
+---
+
+This page shows how you can control the flow of your Dart code using branches:
+
+- `if` statements and elements
+- `if-case` statements and elements
+- `switch` statements and expressions
+
+You can also manipulate control flow in Dart using [loops][], like `for`
+and `while`, or even using [exceptions][], like `try-catch` and `throw`.
+
+## If
+
+Dart supports `if` statements with optional `else` clauses. The condition in
+parentheses after `if` must be an expression that evaluates to a [boolean][]:
+
+<?code-excerpt "misc/lib/language_tour/control_flow.dart (if-else)"?>
+```dart
+if (isRaining()) {
+  you.bringRainCoat();
+} else if (isSnowing()) {
+  you.wearJacket();
+} else {
+  car.putTopDown();
+}
+```
+
+To learn how to use `if` in an expression context, see [Conditional expressions][]
+
+### If-case
+
+Dart `if` statements support `case` clauses followed by a [pattern][]: 
+
+```dart
+if (pair case [int x, int y]) return Point(x, y);
+```
+
+If the pattern (the list pattern `[int x, int y]` in the example above)
+matches the value (`pair`), then the branch (`return Point(x, y)`)
+executes with any variables the pattern defines in scope (`x` and `y`).
+
+Otherwise, the else branch is executed, if there is one:
```

Comment:
**issue:** Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
Otherwise, Dart executes the `else` branch when possible.
```